backend/app/controllers/api/Workspaces.scala

```diff
@@ -241,7 +241,7 @@ class Workspaces(override val controllerComponents: AuthControllerComponents, an
       data <- req.body.validate[MoveItemData].toAttempt
       _ = logAction(req.user, workspaceId, s"Move workspace item. Node ID: $itemId. Data: $data")
 
-      _ <- if (data.newParentId == itemId) Attempt.Left(ClientFailure("Cannot move a workspace item to be under itself")) else Attempt.Right(())
+      _ <- if (data.newParentId.contains(itemId)) Attempt.Left(ClientFailure("Cannot move a workspace item to be under itself")) else Attempt.Right(())
```

Comment:
I think I may have found a genuine mistake, not sure if this code was reached in practice?
@philmcmahon